frontend/src/global_styles/content/work_packages/tabs/_tab_content.sass

```diff
@@ -48,6 +48,10 @@
     padding-bottom: $spot-spacing-0_75
     display: flex
     align-items: center
+    position: sticky
+    top: 0
+    background: $spot-color-basic-white
```

Comment:
why does the header need a background color?

Reply:
Because it's sticky, so the other content will scroll under it

Reply:
do we already have headers in the tabs where the section is scrollable?